tests/default_vars.sh

```diff
@@ -639,6 +653,7 @@ export NFHOUT='6'
 #no high freq fv3 output
 export NFHMAX_HF='-1'
 export NFHOUT_HF='-1'
+export FHZERO=${FDIAG}
```

Comment:
This may cause error as FDIAG would be an array of real values while FHZERO is one integer

Reply:
Ok, I thought I was being clever. I will change it; I need to see why the hera jobs just stopped anyway.

Reply:
Actually @JessicaMeixner-NOAA you need to make this change since it is now your branch we are testing. Thnx.

Reply:
Updated.  Let me know if the fix is incorrect.